packages/opentelemetry-core/package.json

```diff
@@ -20,9 +20,11 @@
     "clean": "rimraf build/*",
     "check": "gts check",
     "precompile": "tsc --version",
-    "compile": "tsc -p .",
+    "compile": "npm run version && tsc -p .",
     "fix": "gts fix",
-    "prepare": "npm run compile"
+    "prepare": "npm run compile",
+    "version": "node scripts/version.js",
```

Comment:
Should this be called something like `version:update` and maybe call the script `update-version.js` or similar?